src/ring_theory/localization/integral.lean

```diff
@@ -349,4 +349,41 @@ begin
       (no_zero_smul_divisors.algebra_map_injective _ _) _ hf₂ }
 end
 
+open_locale non_zero_divisors
+
+variables (R) {S K}
+/-- If the `S`-multiples of `a` are contained in some `R`-span, then `Frac(S)`-multiples of `a`
+are contained in the equivalent `Frac(R)`-span. -/
+lemma ideal_span_singleton_subset {L : Type*}
```

Comment:
should the name mention `algebra_map` or `map`?
```suggestion
lemma ideal_span_singleton_map_subset {L : Type*}
```

Reply:
Most `is_localization` names talk about `map`, so I guess your suggestion is the best.